test/Scrabble/Move/MoveTests.hs

```diff
@@ -0,0 +1,75 @@
+{-# LANGUAGE TemplateHaskell #-}
+
+module Scrabble.Move.MoveTests where
+
+import Test.Framework.Providers.HUnit
+import Test.Framework.TH
+import Test.HUnit
+import TestHelpers
+import Scrabble.Move.Move
+import Scrabble.Board.Board
+import Scrabble.Bag
+
+
+case_create_move =
+    let rack       = Rack $ map fromLetter [O, P, T]
+        (Right wp) = makeWordPut "TOP" Vertical (7,7) []
+        expected = do
+            (board',score) <- wordPut standardValidation newBoard wp dict
+            return $ Move wp score (Rack []) board'
+
+    in createMove newBoard rack wp dict @?= expected
```

Comment:
I think in all of these it's a bit hard, by looking at it, to understand what result is actually expected. Any ideas on how we could clean this up? Hopefully in the code, but if not, even just a comment would help I think.